### PR TITLE
Fix form input for users with dark OS theme

### PR DESCRIPTION
**Pull request by linusg**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

Hey there!

First of all: thanks for this great project!

I noticed the e-mail input does not look great for users with a dark OS theme enabled (I'm one of these). This seems to be a very common issue in web design, as many users of macOS and Windows are not even aware there's a possibility of changing the appearance of OS native widgets; at least on most Linux distributions this is possible (GTK Themes). I even [ranted on Twitter](https://twitter.com/linusgroh/status/979349672602095617) about this once, sadly no great impact...

Long story short: one should NOT expect that the background color of an input will be `#ffffff` or similar! On the other hand, changing the color property to something dark-ish makes it even worse, as it completely makes the input unusable (dark text on dark background).

This PR provides a fix of the color and background color to fit with the overall design.

*I also changed some very very minor things on the fly, I know some maintainers can be picky about this... if so, please feel free to tell me and I'll create a new PR.*

### CODE REVIEW

**Comment by mlinksva** [comment]
I was able to reproduce this on Linux and Firefox (Nightly) with `GTK_THEME="Adwaita:dark" firefox`. The language drop-down in the footer is also affected, though both just look a bit out of place (dark background/light text when the rest of the site is the reverse); the text is visible.

This seems to be the result of a longstanding Firefox [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1283086).

I'm OK with implementing this fix but know almost nothing about design so I'll leave it to someone else to review whether it is merited and that the exact implementation here is the right one.

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @linusg!
